blocklylib-core/build.gradle

```diff
@@ -43,6 +43,28 @@ task sourcesJar(type: Jar) {
     classifier = 'sources'
 }
 
+task getBlockly {
+    def files = [
+            'blockly_compressed.js': 'src/main/assets/blockly_compressed.js',
+            'blocks_compressed.js': 'src/main/assets/blocks_compressed.js',
+            'javascript_compressed.js': 'src/main/assets/generators/javascript_compressed.js',
+            'lua_compressed.js': 'src/main/assets/generators/lua_compressed.js',
+            'php_compressed.js': 'src/main/assets/generators/php_compressed.js',
+            'python_compressed.js': 'src/main/assets/generators/python_compressed.js',
+            'dart_compressed.js': 'src/main/assets/generators/dart_compressed.js',
+            'msg/js/en.js': 'src/main/assets/msg/js/en.js'
```

Comment:
This means every project will be including these other language files, yet very few will need them.

Can we move the non JavaScript generator files to the demo gradle file (and test project gradle file, it looks like), and add a comment in each of the demo activity classes to note the download via the gradle file?

Leave a comment in this file noting that the other languages are available.

Reply:
Created Full and Lite Product Flavors.